apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -69,12 +69,6 @@ export class SendMessage {
     const stepType = command.step?.template?.type;
 
     if (!command.payload?.$on_boarding_trigger) {
-      const usedFilters = shouldRun.conditions.reduce(MessageMatcher.sumFilters, {
```

Comment:
Moving this bit of code that belongs to MessageMatcher simplifies the inheritance of the `sumFilters` functionality.